### PR TITLE
Pandas

### PR DESCRIPTION
**Pull request by kasparvonbeelen**
update pandas

### CODE REVIEW

File: Sessions/3.1 - Working with Tabular Data.ipynb [hunk 1]
```diff
@@ -41,17 +41,13 @@
     }
    },
    "source": [
-    "### Today's Programme: Data Formats and Access\n",
+    "## 1.2 This Session: Analysing Texts Embedded Tabular Data\n",
```
Comment (mcollardanuy) [inline Sessions/3.1 - Working with Tabular Data.ipynb]: Should this be: "embedded **in** tabular data"?

File: Sessions/3.1 - Working with Tabular Data.ipynb [hunk 2]
```diff
@@ -84,2004 +80,2592 @@
...
+   "id": "19d62802",
+   "metadata": {
+    "slideshow": {
+     "slide_type": "slide"
+    }
+   },
+   "outputs": [],
+   "source": [
+    "df_muse_by_genre = df_muse.set_index('genre') # set the genre column as the index of the dataframe\n",
+    "df_muse_by_genre"
```
Comment (mcollardanuy) [inline Sessions/3.1 - Working with Tabular Data.ipynb:1293]: Just one note: `df_muse` hasn't been defined before (except in exercises). Maybe it's worth showing how to load the df?

File: Sessions/3.1 - Working with Tabular Data.ipynb [hunk 3]
```diff
@@ -84,2004 +80,2592 @@
...
+    "- retrieving values by position:\n",
+    "```python \n",
+    "l = ['a','b']\n",
+    "l[1]\n",
+    "```\n",
+    "- retrieving values by key:\n",
+    "```python \n",
+    "d = {'a': 1}\n",
+    "d['a']\n",
+    "\n",
```
Comment (mcollardanuy) [inline Sessions/3.1 - Working with Tabular Data.ipynb]: Something went wrong with the formatting here.

**Review by mcollardanuy — COMMENTED** [review]
Hi @kasparvonbeelen, all good! The only other thing (apart from the comments) is what we've already discussed, the question of maybe downloading the data manually or with wget instead, because of the size.

File: Sessions/3.1 - Working with Tabular Data.ipynb (hunk 1 above)
Comment (kasparvonbeelen) [reply Sessions/3.1 - Working with Tabular Data.ipynb]: Indeed, corrected!

File: Sessions/3.1 - Working with Tabular Data.ipynb (hunk 2 above)
Comment (kasparvonbeelen) [reply Sessions/3.1 - Working with Tabular Data.ipynb:1293]: done!

File: Sessions/3.1 - Working with Tabular Data.ipynb (hunk 3 above)
Comment (kasparvonbeelen) [reply Sessions/3.1 - Working with Tabular Data.ipynb]: corrected :-)

**Review by mcollardanuy — APPROVED** [review]
All good!

**Pull request merged**